L1Trigger/L1TMuonEndCap/src/TrackTools.cc

```diff
@@ -172,4 +172,13 @@ namespace emtf {
     return std::make_pair(max_pattern, max_quality);
   }
 
+  int get_csc_max_slope(int station, int ring, bool useRun3CCLUT_OTMB, bool useRun3CCLUT_TMB) {
```

Comment:
Why is `station` needed here?

Reply:
I added the station there as well to be safe. Currently it's not needed, but I don't know if there will be a station based enabling of CCLUT algorithm in the future. This is also the reason station and ring are there for other functions like `get_csc_max_pattern_and_quality`.